### PR TITLE
SUM-164 SUM-183 | add apply-now-link and left-bar wysiwyg styles

### PR DESCRIPTION
**Pull request by rebeccahongsf**
# READY FOR REVIEW

# Summary
- add apply-now-link and left-bar wysiwyg styles

# Review By (Date)
- When possible

# Review Tasks

## Setup tasks and/or behavior to test

1. Check out this branch
2. Navigate to Course List, Favorites, and Course pages
3. Verify that the Apply Now link block appears.
4. Create a basic page with a WYSIWYG
5. Add the following snippets to the WYSIWYG
```
<a class="apply-now-link">
Ready to dive in? Kick off your application today – let's make things happen!  
</a>
```

```
<div class="left-bar">
<h2>Brace yourself</h2>
<p>Rent can be between $2,500 to $4,000 a month in Palo Alto. We totally get the desire to hunt for a better deal off-campus, but our housing plan is tough to beat.</p>
</div>
```

6. Verify that it appears like the mocks
7. Review code 🪴 

# Associated Issues and/or People
- SUM-164 
- SUM-183

### CODE REVIEW

File: src/components/nodes/pages/summer-course/summer-course-page.tsx [hunk 1]
```diff
@@ -142,7 +143,13 @@ const SummerCoursePage = async ({node, ...props}: Props) => {
       <div className="centered">
         <RelatedCourses objectId={node.id} />
       </div>
-      <div>{/* Apply Now Link */}</div>
+      <div className="centered flex flex-col items-end">
+        <div className="w-full md:w-1/2">
+          <ApplyNowLink href="/courses">
```
Comment (pookmish) [inline src/components/nodes/pages/summer-course/summer-course-page.tsx:148]: I don't believe that's the correct location. The masthead on the current site goes to `/apply-now`

File: src/components/elements/wysiwyg.tsx [hunk 2]
```diff
@@ -53,6 +61,14 @@ const options: HTMLReactParserOptions = {
 
         case "div":
         case "article":
+          if (nodeProps.className && nodeProps.className.includes("left-bar")) {
```
Comment (pookmish) [inline src/components/elements/wysiwyg.tsx:64]: Shouldn't this be on the `div` only?

File: src/components/nodes/pages/summer-course/summer-course-page.tsx (hunk 1 above)
Comment (rebeccahongsf) [reply src/components/nodes/pages/summer-course/summer-course-page.tsx:148]: Ah, I had put `/courses` as a placeholder for now. I'll follow up with Sara in the email thread to confirm the URL destination 😄  Thanks Mike!

File: src/components/elements/wysiwyg.tsx [hunk 3]
```diff
@@ -108,6 +130,12 @@ const options: HTMLReactParserOptions = {
         case "tfoot":
         case "caption":
           return <NodeName {...nodeProps}>{domToReact(children, options)}</NodeName>
+        case "left-vertical":
```
Comment (pookmish) [inline src/components/elements/wysiwyg.tsx:133]: What is the `left-vertical` element?

File: src/components/elements/wysiwyg.tsx [hunk 4]
```diff
@@ -52,6 +60,20 @@ const options: HTMLReactParserOptions = {
           )
 
         case "div":
+          if (nodeProps.className && nodeProps.className.includes("left-bar")) {
+            nodeProps.className = twMerge(
+              nodeProps.className,
+              "max-w-[725px] rs-pb-5 rs-pt-7 rs-pl-1 border-3 border-archway-dark border-opacity-50"
```
Comment (pookmish) [inline src/components/elements/wysiwyg.tsx:66]: Border should only be on the left.
image

File: src/components/elements/wysiwyg.tsx [hunk 5]
```diff
@@ -52,6 +60,20 @@ const options: HTMLReactParserOptions = {
...
+            )
+            return <NodeName {...nodeProps}>{domToReact(children, options)}</NodeName>
+          }
+
+          delete nodeProps.role
+          if (nodeProps.className && !!nodeProps.className.indexOf("media-entity-wrapper")) {
+            return cleanMediaMarkup(domNode)
+          }
+          return <NodeName {...nodeProps}>{domToReact(children, options)}</NodeName>
```
Comment (pookmish) [inline src/components/elements/wysiwyg.tsx:75]: You don't need these because if you don't have a return, it'll go down into the `article` logic.

**Review by pookmish — CHANGES_REQUESTED** [review]

**Review by pookmish — APPROVED** [review]

**Pull request merged**